main.js

```diff
@@ -0,0 +1,92 @@
+    profession: 'Arranger and songwriter/ Bassist/ Music Director At Punjab University.',
+    description: 'VVery Talented, an elite Keyboardist and Bassist. Renowned for his tastful bass lines.',
+  },
+  {
+    img: './assets/photo-3.jpg',
+    fullName: 'Bloch Brothers',
+    profession: 'Arranger and songwriter/ Saxophonist/ Vocal and Choral Director Berklee.',
+    description: 'Recieved Singer and Artist of the year Award in GMA Awards 2020.',
+  },
+  {
+    img: './assets/photo-4.jpg',
+    fullName: 'Bilal Ahmed',
+    profession: 'Music Producer/Music Director at Sydney University.',
+    description: 'Enjoys Music and Perfecting his craft. Widely known for his choral Arrangements and Reharms!',
+  },
+  {
+    img: './assets/photo-5.jpg',
+    fullName: 'Arslan Khan',
+    profession: 'Business Man/Agent for HARMONIX MUSIC SYSTEMS.',
+    description: 'A natural businessperson, and brilliant business mind.',
+  },
+  {
+    img: './assets/photo-6.jpg',
+    fullName: 'Qabeel Khan',
+    profession: 'Youtube Streamer / Gamer',
+    description: 'Known for his Problem solving mind, an Elite Gamer.',
+  },
+
+];
+
+featuredCards.forEach((featureCard) => {
+  const dynamicCardItem = document.createElement('div');
+  dynamicCardItem.classList.add('dynamic_card_item');
+  dynamicCardItem.innerHTML = `
+    <img src="${featureCard.img}" alt="">
+        <div class="dynamic_card_description">
+            <h2 class="full_name">
+                ${featureCard.fullName}
+            </h2>
+             <p class="dynamic_card_profession">
+                ${featureCard.profession}
+            </p>
+            <div class="dynamic_card_sepreator">
+            </div>
+            <p class="dynamic_card_details">
+                ${featureCard.description}
+            </p> 
+            `;
+  featuredSection.appendChild(dynamicCardItem);
+});
+
+// mobile menu
+
+const hamburger = document.querySelector('.fa-bars');
+const menu = document.createElement('div');
+menu.classList.add('menu');
+menu.classList.add('hide');
+menu.innerHTML = `<div class="cross">
+            <i class="fas fa-times"></i>
+        </div>
+        <div class="pages">
+            <a href="index.html" class="page_items" onclick="closeMenu()">Home</a>
+            <a href="about.html" class="page_items" onclick="closeMenu()">About</a>
+            <a href="#" class="page_items" onclick="closeMenu()">Program</a>
+            <a href="#" class="page_items" onclick="closeMenu()">Join</a>
+            <a href="#" class="page_items" onclick="closeMenu()">Sponsor</a>
+            <a href="#" class="page_items" onclick="closeMenu()">News</a>
+        </div>`;
+document.body.appendChild(menu);
+const closeBtn = document.querySelector('.fa-times');
+
+function showMenu() {
+  return menu.classList.remove('hide');
+}
+function closeMenu() {
+  return menu.classList.add('hide');
+}
+
+hamburger.addEventListener('click', showMenu);
+closeBtn.addEventListener('click', closeMenu);
```

Comment:
- [ ] Your project looks very nice and well done :+1:  but in Javascript, your code is not [DRY](https://devinduct.com/blogpost/22/javascript-clean-code-best-practices). DRY, which stands for `don't repeat yourself,` is a principle of software development that aims at reducing the repetition of patterns and code duplication in favour of abstractions and avoiding redundancy. :slightly_smiling_face: 
- [ ] Kindly extracted the pages data into an array for easier management and rendering.  :+1: 
- [ ] Please used `map()` and `join()` to generate the pages HTML dynamically. :smile: 
- [ ] Combined the `showMenu()` and `closeMenu()` functions into a single `toggleMenu()` function that toggles the `hide` class on the `menu` element. :slightly_smiling_face: 
- [ ] Changed the event listener for the close button to query the menu element instead of the document. :+1: 
- [ ] Check below my suggestions :point_down: 

```suggestion
const hamburger = document.querySelector('.fa-bars');
const menu = document.createElement('div');
const pages = [
  { href: 'index.html', label: 'Home' },
  { href: 'about.html', label: 'About' },
  { href: '#', label: 'Program' },
  { href: '#', label: 'Join' },
  { href: '#', label: 'Sponsor' },
  { href: '#', label: 'News' },
];
const pagesHtml = pages.map(
  ({ href, label }) =>
    `<a href="${href}" class="page_items" onclick="closeMenu()">${label}</a>`
).join('');
menu.classList.add('menu', 'hide');
menu.innerHTML = `
  <div class="cross">
    <i class="fas fa-times"></i>
  </div>
  <div class="pages">
    ${pagesHtml}
  </div>
`;
document.body.appendChild(menu);

function toggleMenu() {
  menu.classList.toggle('hide');
}

hamburger.addEventListener('click', toggleMenu);
menu.querySelector('.fa-times').addEventListener('click', toggleMenu);
```